.github/workflows/wiki_label.yml

```diff
@@ -0,0 +1,53 @@
+name: 'PR Wiki Label'
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  wiki-label:
+    runs-on: ubuntu-latest
+    if: github.event.issue.pull_request && github.event.comment.body == '!wiki_label'
+    permissions:
+      contents: read
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Check commenter authorization
+        id: check-auth
+        run: |
+          AUTHORIZED_USERS=("SynthTwo")
```

Comment:
We can make this into a repo variable instead. So HCs can tweak it easier without having to do a code change. I've made one with the name `WIKI_MANAGERS`
https://docs.github.com/en/actions/learn-github-actions/variables#creating-configuration-variables-for-a-repository
Then we can use the following to get the list from the string value:
https://stackoverflow.com/questions/75391000/define-list-of-elements-in-a-github-action-variable
And check if the username is in the list of allowed usernames
https://stackoverflow.com/questions/65653291/conditional-to-check-if-a-string-is-included-in-a-list

Reply:
I have made the changes, but I will have to test this again locally via my fork to make sure it works properly.

Reply:
We are now using the WIKI_MANAGERS repository variable for this. Testing to be done locally.

Reply:
This has now been tested and updated. 
Repository variable instructions and updating to be handled internally.